README.md

```diff
@@ -87,8 +90,12 @@ If the SDKs will be installed on different JVMs, then you would need to instanti
 ```java
     /* Now create a Wavefront Jersey Filter which you can add to your 
     * Jersey based (Dropwizard, Springboot etc.) application  */
-    WavefrontJerseyFilter wfJerseyFilter = new WavefrontJerseyFilter(wfJerseyReporter, 
-        applicationTags);
+    WavefrontJerseyFilter wfJerseyFilter = new WavefrontJerseyFilter.Builder(
+        wfJerseyReporter, applicationTags).build();
+    
+    /* If you want to send tracing data and have WavefrontTracer configured */
+    WavefrontJerseyFilter wfJerseyFilter = new WavefrontJerseyFilter.Builder(
```

Comment:
Invoke withTracer(tracer) on the WavefrontJerseyFilter.Builder on this line.
Keep the comment -
/* If you want to send tracing data and have WavefrontTracer configured */